store/tikv/2pc.go

```diff
@@ -666,44 +669,60 @@ func sendTxnHeartBeat(bo *Backoffer, store *tikvStore, primary []byte, startTS,
 	}
 }
 
+// checkAsyncCommit checks if async commit protocol is available for current transaction commit, true is returned if possible.
+func (c *twoPhaseCommitter) checkAsyncCommit(ctx context.Context) bool {
+	const asyncCommitKeysLimit = 256
+	if c.connID > 0 && config.GetGlobalConfig().TiKVClient.EnableAsyncCommit && len(c.mutations.keys) <= asyncCommitKeysLimit {
```

Comment:
when will `c.ConnID` will be 0~?

Reply:
The system session running background works.

Reply:
Why do we need to bypass the internal SQLs?

Reply:
To make the tests easier by now, maybe we could remove this constraint if it's stable.